elastic/kibana
# [Synthetics] Getting started use url as monitor name

## Summary

Fixes https://github.com/elastic/kibana/issues/142982

Getting started use url as monitor name.

## How to test
1. Create a new Kibana Space, or start a new Kibana instance so that you start with 0 synthetics monitors.
2. Enable Synthetics from Kibana -> Advanced Settings and goto Synthetics app.
3. Make sure you land on the Getting Started page with a simple form to create the first monitor.
4. Enter a site url and choose a location.
5. Click **Create monitor** button and make sure it redirects you to Monitors page.
6. Make sure the name of the monitor is the url of the site you entered.
7. Delete all monitors and make sure it redirects you back to Getting Started page.

_Note_: You can also use this [deployment](https://p.elstc.co/paste/G9uhicYJ#+cBBt9E7XEr+89uDphj7eqvo86yQN-TvRVykRhkX0O4) to test (if it's still live).
